Bug: --mcp fails on Windows due to double drive letter in path (C:\C:\...)

## Bug Description

Running `bb-browser --mcp` on Windows fails with `MODULE_NOT_FOUND` error because the resolved path to `mcp.js` contains a duplicated drive letter prefix (`C:\C:\...`).

## Error Message

```
Error: Cannot find module 'C:\C:\Users\Admin\AppData\Local\npm-cache\_npx\eaa85051a9929d4c\node_modules\bb-browser\dist\mcp.js'
    at Function._resolveFilename (node:internal/modules/cjs/loader:1365:15)
    ...
```

## Root Cause

In `packages/bb-browser/src/cli.ts` (compiled to `dist/cli.js`), the `--mcp` handler constructs the path to `mcp.js` using:

```js
const mcpPath = new URL("./mcp.js", import.meta.url).pathname;
```

On Windows, `new URL(...).pathname` returns a POSIX-style path with a leading slash like `/C:/Users/Admin/...`. When Node.js later resolves this, it incorrectly produces `C:\C:\Users\Admin\...` (double drive letter), which is an invalid path.

## Suggested Fix

Use `fileURLToPath()` from the `node:url` module instead:

```js
import { fileURLToPath } from "node:url";
const mcpPath = fileURLToPath(new URL("./mcp.js", import.meta.url));
```

`fileURLToPath()` correctly handles the `file://` URL-to-path conversion on all platforms, including Windows.

## Environment

- OS: Windows 10 (10.0.26200)
- Node.js: v22.18.0
- npm: 11.5.2
- bb-browser: latest (installed via `npx -y bb-browser@latest --mcp`)

## Workaround

Users can bypass the issue by invoking the MCP server binary directly:

```powershell
npx -y bb-browser-mcp@latest
```

or by running `mcp.js` with node directly:

```powershell
node node_modules/bb-browser/dist/mcp.js
```

--- a/packages/cli/src/index.ts
+++ b/packages/cli/src/index.ts
@@ -15,6 +15,7 @@
  *   --json                    以 JSON 格式输出
  */
 
+import { fileURLToPath } from "node:url";
 import { openCommand } from "./commands/open.js";
 import { snapshotCommand } from "./commands/snapshot.js";
 import { clickCommand } from "./commands/click.js";
@@ -224,7 +225,7 @@ async function main(): Promise<void> {
   }
 
   if (process.argv.includes("--mcp")) {
-    const mcpPath = new URL("./mcp.js", import.meta.url).pathname;
+    const mcpPath = fileURLToPath(new URL("./mcp.js", import.meta.url));
     const { spawn } = await import("node:child_process");
     const child = spawn(process.execPath, [mcpPath], { stdio: "inherit" });
     child.on("exit", (code) => process.exit(code ?? 0));

